install_config/configuring_aws.adoc

```diff
@@ -18,6 +18,56 @@ volumes as persistent storage] for application data. After AWS is configured
 properly, some additional configurations will need to be completed on the
 {product-title} hosts.
 
+[[configuring-aws-permissions]]
+== Permissions
+Configuring AWS for {product-title} requires the following permissions on:
+
+* Master
```

Comment:
I would keep the intro sentence but rework it to say:
Configuring AWS for {product-title} requires the following permissions.

Then, instead of using a bullet point to label the tables, I would use:

. Master Permissions

and

.Node Permissions

Reply:
Thanks, I think you mean `.Master Permissions` (table heading) and not `. Master Permissions` (numbered list)